twitter/Models/Tweet.h

```diff
@@ -0,0 +1,33 @@
+//
+//  Tweet.h
+//  twitter
+//
+//  Created by Joey R. Hernandez Perez on 6/29/20.
+//  Copyright © 2020 Emerson Malca. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "User.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Tweet : NSObject
+
+@property (nonatomic, strong) NSString *idStr; // For favoriting, retweeting & replying
```

Comment:
I can figure out that this is ID string but a few things to consider. 

1) is this your id or twitters' id? 
2) is there a good reason to abbreviate either word?